### ISSUE TITLE
Loading swap files errors telescope/actions/init.lua:127: Vim(edit):E325: ATTENTION

### ISSUE THREAD

**Issue by rockerBOO** [nvim-telescope/telescope.nvim#445]
### Description

Opening files under swap, error, when recovering from telescope. The error does not seem to prevent me from opening the file after pressing enter. Lower priority but annoying for anyone using swap files.

**Expected Behavior**
After recovering the file from swap, it should open the file without erroring.

**Actual Behavior**
```
Found a swap file by the name "~/.config/local/share/nvim/swap//%home%rockerboo%.dotfiles%config%nvim%lua%plugin%tel
escope.lua.swp"
          owned by: rockerboo   dated: Tue Jan 19 14:39:52 2021
         file name: ~rockerboo/.dotfiles/config/nvim/lua/plugin/telescope.lua
          modified: no
         user name: rockerboo   host name: Monolith
        process ID: 32451 (STILL RUNNING)
While opening file "/home/rockerboo/.config/nvim/lua/plugin/telescope.lua"
             dated: Tue Jan 19 13:57:13 2021

(1) Another program may be editing the same file.  If this is the case,
    be careful not to end up with two different instances of the same
    file when making changes.  Quit, or continue with caution.
(2) An edit session for this file crashed.
    If this is the case, use ":recover" or "vim -r /home/rockerboo/.config/nvim/lua/plugin/telescope.lua"
    to recover the changes (see ":help recovery").
    If you did this already, delete the swap file "/home/rockerboo/.config/local/share/nvim/swap//%home%rockerboo%.d
otfiles%config%nvim%lua%plugin%telescope.lua.swp"
    to avoid this message.

Swap file "~/.config/local/share/nvim/swap//%home%rockerboo%.dotfiles%config%nvim%lua%plugin%telescope.lua.swp" alre
ady exists!
[O]pen Read-Only, (E)dit anyway, (R)ecover, (Q)uit, (A)bort:
```

```
"~/.config/nvim/lua/plugin/telescope.lua" 78 lines, 2120 characters
Using swap file "~/.config/local/share/nvim/swap//%home%rockerboo%.dotfiles%config%nvim%lua%plugin%telescope.lua.swp
"
Original file "~/.dotfiles/config/nvim/lua/plugin/telescope.lua"
Recovery completed. Buffer contents equals file contents.
You may want to delete the .swp file now.


E5108: Error executing lua ...cker/start/telescope.nvim/lua/telescope/actions/init.lua:127: Vim(edit):E325: ATTENTION
```

### Details

<details><summary>Reproduce</summary>

1. nvim -nu test.vim
2. open another nvim with a file open
2. :Telescope find_files to find the file open in the previous nvim
3. Swap error
</details>

<details><summary>Environment</summary>

- nvim --version output: NVIM v0.5.0-dev+1037-gd6d4e3d1a
- Operating system: Arch linux
- Telescope commit: 6461b07

</details>

<details><summary>Configuration</summary>
<p>

```viml
set nocompatible hidden laststatus=2

set swapfile=true

if !filereadable('/tmp/plug.vim')
  silent !curl --insecure -fLo /tmp/plug.vim
    \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
endif

source /tmp/plug.vim
call plug#begin('/tmp/plugged')
Plug 'nvim-lua/popup.nvim'
Plug 'nvim-lua/plenary.nvim'
Plug 'nvim-telescope/telescope.nvim'
call plug#end()

autocmd VimEnter * PlugClean! | PlugUpdate --sync | close
lua << EOF

require('telescope').setup{
  defaults = {
    vimgrep_arguments = {
      'rg',
      '--color=never',
      '--no-heading',
      '--with-filename',
      '--line-number',
      '--column',
      '--smart-case'
    },
    prompt_position = "bottom",
    prompt_prefix = ">",
    selection_strategy = "reset",
    sorting_strategy = "descending",
    layout_strategy = "horizontal",
    layout_defaults = {},
    file_ignore_patterns = {},
    shorten_path = true,
    winblend = 0,
    width = 0.75,
    preview_cutoff = 120,
    results_height = 1,
    results_width = 0.8,
    border = {},
    borderchars = { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
    color_devicons = true,
    use_less = true,
  }
}
EOF

```

**Comment by Conni2461** [comment]
I think that is `vim.cmd` telling us that something went wrong. We could `pcall` actions.lua:127 to solve that

**Comment by rockerBOO** [comment]
I can try that and report back.

**Comment by AlejandroVaqueraLopez** [comment]
To solve this problem I just deleted the plugin.vim.swap file in C:\Users\personal\AppData\Local\nvim-data\swap hope it works for you

**Comment by gavynriebau** [comment]
I was encountering the same issue, but with a slightly different error message and have raised a PR to fix it here: #1247 

You were seeing the error message:

> E5108: Error executing lua ...cker/start/telescope.nvim/lua/telescope/actions/init.lua:127: Vim(edit):E325: ATTENTION

While I was seeing:

> E5108: Error executing lua ...acker/start/telescope.nvim/lua/telescope/actions/set.lua:134: Vim(edit):E325: ATTENTION

I looked in the file `telescope/actions/init.lua` to see how they had fixed the problem in that file and saw they wrapped calls to `vim.cmd` so I did the same thing to fix the issue in `telescope/actions/set.lua`.

Worth noting that to reproduce the error I was following almost the same steps as you but I chose the option "(E)dit anyway" while I believe you chose "(R)ecover".

**Comment by Conni2461** [comment]
There is a difference in the file because we moved that code. This issue was originally opened in January. A lot happened since then. Thanks for opening a PR :)

This issue should be fixed. I am closing it. If its still reproduce able feel free to reopen :)
